src/style.css

```diff
@@ -1 +1,72 @@
 /* Styles go here */
+
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+}
+
+.nav__item {
+  margin: 0 10px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  height: 60px;
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  color: #000;
+  font-family: Roboto , sans-serif;
+  text-transform: uppercase;
+  line-height: 14px;
+  font-size: 12px;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  /* width: 37px; */
+  height: 4px;
+  background-color: #00acdc ;
```

Comment:
you typed the #00acdc 3 times
but what if we decide to use another color?
we would need to find all the places we use it and change
if you use CSS variable you can change the color in 1 place

Reply:
You are right , ill make a var next time , thanks.